src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -184,6 +189,27 @@ public IEnumerable<IGraphType> All()
             }
         }
 
+        private void AddType(Type builtInGraphType)
```

Comment:
I would suggest perhaps a more distinct name like `AddBuiltInType`?  Too closely named to the other `AddType` methods I think.

I like this direction.  👍🏼

Reply:
Good idea! - done. 👍 

Let me know if you have any other comments.  It's a pretty simple enhancement, and will, I believe, work great all the way 'round.

Reply:
I already intervened in this PR and changed a lot. This method will no longer exist. I will update the PR and describe the changes in the next hour.

Reply:
Resolving conflicts 😞